README.md

```diff
@@ -1,11 +1,222 @@
 envoy-go-ldap-auth
 ==================
 
-This is a simple LDAP auth filter for envoy written in go.
+This is a simple LDAP auth filter for envoy written in go. Only requests that pass the LDAP server's authentication will be proxied to the upstream service.
 
+During this process, we can optimize the system by implementing user information caching with a duration defined by `config.cache_ttl`. This approach will help reduce the frequency of LDAP server access. If it is set to 0, caching is disabled by default.
 
-Status
-======
+In terms of caching, we utilize [bigcache](https://github.com/allegro/bigcache), which demonstrates exceptional performance in the evicting cache domain.
```

Comment:
I'm not sure if it's worth to introduce such a cache component, since the cache should not be too big?
do you have some benchmarks? in different cases.
what's the traefik's choice?

Reply:
In general, we should be cautious about introducing third libraries.

Reply:
fine. I will update this later.

Reply:
Traefik does not currently have this optimization. Perhaps for the MVP version, it can be omitted for now.

Reply:
okay, let's keep it simple. remove this optimization could be better.

Reply:
Done.

Reply:
this doc also need update

Reply:
Done.